Option 'declarationDir' cannot be specified without specifying option 'declaration' or option 'composite'

I'm getting this error

```
    Option 'declarationDir' cannot be specified without specifying option 'declaration' or option 'composite'.
    Option 'declarationMap' cannot be specified without specifying option 'declaration' or option 'composite'.
```
when passing a path to this `tsconfig.json` only; it uses `declarationMap` whereas other tsconfigs in the application do not and parsing types into schema works fine for them.

```
{
  "extends": "../../tsconfig-global.json",
  "compilerOptions": {
    "lib": ["scripthost", "es2019"],
    "outDir": "./dist",
    "types": ["node", "jest"],
    "declaration": true,
    "declarationMap": true,
    "declarationDir": "./dist",
    "plugins": [{ "transform": "@zerollup/ts-transform-paths" }]
  },
  "include": ["src/"]
}
```

This seems to be an error from the typescript compiler, but "declaration" is present already; so I'm guessing this is not passed through verbatim for some reason?

If I remove all three directives from my tsconfig, it works fine.

This turns out to be really trivial to fix, I think all that needs to change is in `program.ts` @ line 34 add:

```
        delete parseResult.options.declarataionDir;
        delete parseResult.options.declarataionMap;
```

I pulled down the source repo to make a PR, but I'm getting a type error in master and can't build. Not sure why.

```
src/NodeParser/TupleNodeParser.ts:17:18 - error TS2339: Property 'elements' does not exist on type 'TupleTypeNode'.
17             node.elements
                    ~~~~~~~~
src/NodeParser/TupleNodeParser.ts:18:23 - error TS7006: Parameter 'item' implicitly has an 'any' type.
18                 .map((item) => {
                         ~~~~
Found 2 errors.
```

--- a/factory/program.ts
+++ b/factory/program.ts
@@ -31,6 +31,8 @@ function loadTsConfigFile(configFile: string) {
         delete parseResult.options.outDir;
         delete parseResult.options.outFile;
         delete parseResult.options.declaration;
+        delete parseResult.options.declarationDir;
+        delete parseResult.options.declarationMap;
 
         return parseResult;
     } else {

